ncrypt/crypt.c

```diff
@@ -1209,6 +1209,12 @@ int mutt_protected_headers_handler(struct Body *b_email, struct State *state)
 
   if (c_devel_security)
   {
+    if (b_email->mime_headers->message_id &&
```

Comment:
I'm wondering if this is necessary.  The others don't seem to have it.  Anyone can comment?

Reply:
I don't remember if I added it out of inertia, or if I needed it.

Reply:
Ahh, yes, the subject right above also does the same thing.  It seems we need it.